config/locales/en/admin.yml

```diff
@@ -693,6 +693,7 @@ en:
           submit_button: Create question
         table:
           title: Title
+          description: "Description"
```

Comment:
We could also remove this line if we used `human_attribute_name` in the table header, or if we didn't show the description in the questions table. Less work for our translators :smile:.